src/modules/fundingManager/bondingCurveFundingManager/ToposBondingCurveFundingManager.sol

```diff
@@ -132,13 +134,21 @@ contract ToposBondingCurveFundingManager is
         liquidityPool = ILiquidityPool(_liquidityPool);
         // Set formula contract
         formula = IToposFormula(bondingCurveProperties.formula);
+        // Set capital required
         _setCapitalRequired(bondingCurveProperties.capitalRequired);
+        // Set current seize to MAX_SEIZE
+        _setSeize(bondingCurveProperties.seize);
+        // Set initial
```

Comment:
Should this say `/// Set initial base price multiplier` or is this intentional?

Reply:
You're right, did not finish the sentence